web_hosting_manager/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # SAFE Hosting Manager App Change Log
 
+## [0.4.4]
```

Comment:
@hunterlester , let's please start using the new template for the changelog, as an example you can use this: https://github.com/maidsafe/beaker-plugin-safe-app/commit/f15afd875d011663e512e2e60aaee862f41e06e4#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR3
Eventually we may not only automate this but also have scripts to verify its correctness.